index.md

```diff
@@ -17,24 +17,159 @@ redirect_from:
 Thank you!
 ==========
 
-The [DNS flag day 2019](/2019/) was very successful event and the Internet community
-worked together and fixed problems which were causing delays and other
-problems for Internet users worldwide. We would like to thank to all
+The [DNS flag day 2019](/2019/) was very successful event and the Internet
```

Comment:
```
The [2019 DNS flag day](/2019/) was a very successful event. The Internet
community worked together and fixed problems which were causing delays and
other problems for Internet users worldwide. We would like to thank all
operators who cooperated and helped to make Internet a better place.
```

Reply:
I can't really see any text change here, beside line break, so I skipped it.

Reply:
It splits a sentence in two and adds some missing words.

Reply:
```suggestion
The [2019 DNS flag day](/2019/) was a very successful event. The Internet
```